weblate/templates/trans/change_list.html

```diff
@@ -38,12 +38,8 @@
 </div>
 {% endif %}
 
-{% include "paginator.html" %}
-
 {% include "last-changes-content.html" with last_changes=object_list %}
 
-{% include "paginator.html" %}
```

Comment:
The paginator should stay here.

Also, you've removed only the rendering of the paginator, but the results are still paginated, so that would not work correctly...